src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services_new_in_initializer.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
+use Symfony\Component\DependencyInjection\Exception\LogicException;
+use Symfony\Component\DependencyInjection\Exception\RuntimeException;
+use Symfony\Component\DependencyInjection\ParameterBag\FrozenParameterBag;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+
+/**
+ * This class has been auto-generated
+ * by the Symfony Dependency Injection Component.
+ *
+ * @final
+ */
+class ProjectServiceContainer extends Container
+{
+    private $parameters = [];
+
+    public function __construct()
+    {
+        $this->services = $this->privates = [];
+        $this->methodMap = [
+            'foo' => 'getFooService',
+        ];
+
+        $this->aliases = [];
+    }
+
+    public function compile(): void
+    {
+        throw new LogicException('You cannot compile a dumped container that was already compiled.');
+    }
+
+    public function isCompiled(): bool
+    {
+        return true;
+    }
+
+    public function getRemovedIds(): array
+    {
+        return [
+            'Psr\\Container\\ContainerInterface' => true,
+            'Symfony\\Component\\DependencyInjection\\ContainerInterface' => true,
+        ];
+    }
+
+    /**
+     * Gets the public 'foo' shared autowired service.
+     *
+     * @return \Symfony\Component\DependencyInjection\Tests\Fixtures\NewInInitializer
+     */
+    protected function getFooService()
+    {
+        return $this->services['foo'] = new \Symfony\Component\DependencyInjection\Tests\Fixtures\NewInInitializer(bar: 234);
```

Comment:
Here is the target: dumping a named argument is the only way to skip the previous `$foo = new \stdClass()` argument.

Reply:
Having such a comment written around the (nontrivial) code added in `AutowirePass` would probably be helpful